src/hb-ot-os2-table.hh

```diff
@@ -93,10 +93,56 @@ struct OS2
 {
   enum { tableTag = HB_OT_TAG_OS2 };
 
+  inline bool has_data () const { return this != &Null (OS2); }
```

Comment:
We typically don't like comapring to Null as a way to detect emptyness.  This has legitimate reasons (there might be different null polls for different .so objects, or even files...).  If there's a better way, use it.  Why not the version number, like we do in all other tables?!

Reply:
"The version number for the OS/2 table: 0x0000 to 0x0005.", something more reliable please?

Reply:
Then we need to add a custom null table for it.   Umm. Keep it as is.  I'll clean up at some point when I'll go over all such checks to Null..